src/vcf/fixer.cpp

```diff
@@ -235,7 +235,7 @@ namespace ebi
     void Fixer::visit(DuplicationError &error)
     {
         BOOST_LOG_TRIVIAL(debug) << "Line " << line_number << ": Fixing duplicate: removing variant: "
-                                 << std::string{line->begin(), line->end()};
+                                 << std::string{line->begin(), line->end()-1};
```

Comment:
This displayed an extra end line previously. The `line` contained one and another one was added by the logger.